config/inactive_offices.yml

```diff
@@ -238,7 +238,6 @@ production:
     - 2M241U
     - 2M247A
     - 2P619D
-    - 2Q248M
```

Comment:
The ticket asks to check if this office code is active. It was in both the active and inactive office code list and was in both lists when the office codes [were being updated](https://dsdmoj.atlassian.net/browse/CRIMAPP-244). My assumption now is that we want it to be active based on the current ticket, so i have removed from the inactive list but we can also wait for confirmation from fleur

Reply:
Agree, that's my understanding from the ticket too